### DISCUSSION TITLE
Pybricks code EV3

### DISCUSSION THREAD

**Discussion by Barry-Williams** [pybricks/support#732] [Q&A] [answered]
Hi, I am wondering if it is or will be possible to code an EV3 with the code.pybricks.com or beta.pybricks.com in the future.
I am a teacher using EV3 and the VS Code work flow. However the website workflow would be nice for both student laptops (BYOD) and school laptops (that they will not install VS Code on due to locking down CMD)

**Comment by dlech** [comment, marked as answer]
Pybricks Code requires Bluetooth Low Energy but EV3 only has Bluetooth Classic, so it isn't technically possible to do that right now.
